lib/fast_stats/mean.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# A class for managing the arithmetic and geometric means
+# of positive numbers
+module FastStats
+  class Mean
+
+    DEFAULT_ROUND = 15
+    attr_reader :name, :sum, :log_sum, :n
+
+    def initialize(name:)
+      @name = name
+      @sum = 0.0
+      @n = 0
+      @log_sum = 0.0
+      @log_n = 0
+    end
+
+    def add(val)
+      throw ArgumentError.new "#add, val must be > 0" if val < 0
```

Comment:
Either the comment should be `>=` or the logic should be `<=`, right?

This misses the `0` case, at least as documented.